### PR TITLE
Django forms

### PR DESCRIPTION
**Pull request by bloodearnest**

### CODE REVIEW

File: acceptable/__main__.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def main():
         cli_args = parse_args()
         sys.exit(cli_args.func(cli_args))
     except Exception as e:
+        raise
```
Comment (tomwardill) [inline acceptable/__main__.py]: Will this not prevent the sys.exit from running?

File: acceptable/__main__.py [hunk 2]
```diff
@@ -53,6 +54,12 @@ def parse_args(raw_args=None, parser_cls=None, stdin=None):
     metadata_parser.add_argument('modules', nargs='+')
     metadata_parser.set_defaults(func=metadata_cmd)
 
+    django_parser = subparser.add_parser(
+        'django-metadata',
+        help='Import django project and print extracted metadata in json')
+    django_parser.add_argument('django_app', nargs='*')
```
Comment (tomwardill) [inline acceptable/__main__.py]: You have 'project' in the help and 'app' in the argument name.

File: acceptable/__main__.py (hunk 1 above)
Comment (bloodearnest) [reply acceptable/__main__.py]: Gah, that was debugging code left in, will remove, thanks

File: acceptable/__main__.py (hunk 2 above)
Comment (bloodearnest) [reply acceptable/__main__.py]: I don't need that arg anyway, will remove

**Pull request merged**